pkg/metrics/metrics.go

```diff
@@ -229,6 +250,16 @@ func (hm *hcoMetrics) IsHCOMetricHyperConvergedExists() (bool, error) {
 	return val == HyperConvergedExists, nil
 }
 
+// SetHCOMetricSystemHealthStatus sets the gauge to status
+func (hm *hcoMetrics) SetHCOMetricSystemHealthStatus(status float64) error {
+	return hm.SetMetric(HCOMetricSystemHealthStatus, nil, status)
+}
+
+// GetHCOMetricSystemHealthStatus returns current value of gauge. If error is not nil than value is undefined
```

Comment:
```suggestion
// GetHCOMetricSystemHealthStatus returns current value of gauge. If error is not nil then value is undefined
```

Reply:
Are you sure? I think it's not a matter of time (then), but more "if x happens, than y happens"

Reply:
IMO, "than" is used when comparing things:
- sooner rather than later
- x is greater than y

In this case, I feel (correct me if I'm wrong) that we aren't comparing anything. It's more about what happens ("value is undefined") when a certain condition ("error is not nil") is met.

Apologies for making this about English language rather than Go. 😄

Reply:
I guess you're right, thanks. Updated